src/components/views/rooms/ReplyTile.tsx

```diff
@@ -125,15 +114,15 @@ export default class ReplyTile extends React.PureComponent<IProps> {
         }
 
         const msgtypeOverrides = {
-            "m.image": MImageReplyBody,
+            [MsgType.Image]: MImageReplyBody,
             // We don't want a download link for files, just the file name is enough.
-            "m.file": TextualBody,
+            [MsgType.File]: TextualBody,
             "m.sticker": TextualBody,
```

Comment:
`m.sticker` is an eventType, not a msgType

Reply:
I was thinking there was something weird going on :D